plugin/data_masking/src/udf/udf_registration.cc

```diff
@@ -0,0 +1,139 @@
+/* Copyright (c) 2018, 2019 Francisco Miguel Biete Banon. All rights reserved.
```

Comment:
This file (and some others) seems to be the same as in the group replication plugin?

Please retain the Oracle copyright for files copied from Oracle plugins, and include Oracle in the plugin authors.

Reply:
May I know why this conversation is marked resolved? Isn't the copyright requirement not applicable anymore?

Reply:
Sorry, I forgot to commit-push